capstone/src/main/webapp/youtubeScript.js

```diff
@@ -0,0 +1,137 @@
+const ATTRIBUTES_BY_LANGUAGE = {
```

Comment:
I agree - it might be handy to just leave the `script.js` file unchanged in this PR so it does not interfere with the text analyzer, but exclusively use `youtubeScript.js` for this webpage

Reply:
I've added back script.js in  "address nits"

Reply:
If this `ATTRIBUTES_BY_LANGUAGE` is identical to what appears in `script.js`, one option might be to create a new file, maybe called `perspective_consts.js` or even just `perspective.js`, that could be loaded on both the youtube and non-youtube pages (above where scripts.js and youtubeScript.js are loaded).  That way you can be certain these are only declared once per webpage, and all pages will use the same values.